ETC.tsv

```diff
@@ -14772,62 +14772,62 @@ ETC_20151001_014773	{img emoticon_0010 30 30} Surprised
-ETC_20151001_014812	
-ETC_20151001_014813	
-ETC_20151001_014814	
-ETC_20151001_014815	
-ETC_20151001_014816	
-ETC_20151001_014817	
-ETC_20151001_014818	
-ETC_20151001_014819	
-ETC_20151001_014820	
-ETC_20151001_014821	
-ETC_20151001_014822	
-ETC_20151001_014823	
-ETC_20151001_014824	
-ETC_20151001_014825	
-ETC_20151001_014826	
-ETC_20151001_014827	
-ETC_20151001_014828	
-ETC_20151001_014829	
-ETC_20151001_014830	
-ETC_20151001_014831	
-ETC_20151001_014832	
+ETC_20151001_014777	A conditional sentence selected should be true at all times.
+ETC_20151001_014778	A conditional sentence selected should be true at all times. Please refer to follow information.
+ETC_20151001_014779	A conditional sentence selected should be true at all times. Close the program.
+ETC_20151001_014780	Failed to reset the table. Please refer to the previous log about the following information and file load failure.
+ETC_20151001_014781	Failed to reset the server. Close the program.
+ETC_20151001_014782	Nexon AuthCheck has failed. Please check the error code table that corresponds with the error code of authCheck.h.
+ETC_20151001_014783	Forwared follow errors or logs to client.
+ETC_20151001_014784	Failed to memory Allocate on Binary. Check Binary size appointment.
+ETC_20151001_014785	Happens Packet Buffer Overflow.
+ETC_20151001_014786	Deleted previouse object due to duplicated GUID.
+ETC_20151001_014787	Unable to find the item in the inventory.
+ETC_20151001_014788	The client class already exists.
+ETC_20151001_014789	The client class does not exist.
+ETC_20151001_014790	Invalid Slot ID
+ETC_20151001_014791	Incorrect Equip Slot
+ETC_20151001_014792	You tried to transmit too many items to the Auction House.
+ETC_20151001_014793	Auction House Logic raised an error.
+ETC_20151001_014794	Failed to create IES object.
+ETC_20151001_014795	IES object's GUID is duplicated.
+ETC_20151001_014796	Can't load Barrack PC. Please check the information.
+ETC_20151001_014797	Failed to save the property on IES object.
+ETC_20151001_014798	Tried to access the letter form with number form property.
+ETC_20151001_014799	Tried to access the number form with letter form property.
+ETC_20151001_014800	Invalid coordinates. Failed to create Path Agent.
+ETC_20151001_014801	Invalid handle.
+ETC_20151001_014802	The chatting group does not exist.
+ETC_20151001_014803	Failed to Decompress the Packet data.
+ETC_20151001_014804	Failed to bring the property from IES object.
+ETC_20151001_014805	Tried to use the Zombie object.
+ETC_20151001_014806	Call Stack to be set up recording on a cycle. Please refer to the different log information saved at the same time.
+ETC_20151001_014807	Failed to initialization Nexon SSO.
+ETC_20151001_014808	Failed to initialization library. Please refer to the follow information, file load failure and previous logs.
+ETC_20151001_014809	Failed to allocate the memory.
+ETC_20151001_014810	The error of the mission room appeared. Please refer to follow information.
```

Comment:
A mission room-related error has occurred. Please check the following information.